tests/suites/cloud/test-app/containerB/Dockerfile.template

```diff
@@ -0,0 +1,3 @@
+FROM balenalib/%%BALENA_ARCH%%-alpine
```

Comment:
Do tag this image.

Reply:
Seeing what you are using these containers for, I recommend building using the run variant. Go even smaller.